web-animations-2/Overview.bs

```diff
@@ -1066,13 +1066,41 @@ condition:
 > *   the <a>animation effect</a> has a <a>parent group</a> and the
 >     <a>parent group</a> is <a>current</a>.
 
-Add the following definition to the list used in determining the phase of
+Add the following definitions to the list used in determining the phase of
 an <a>animation effect</a>.
 
 :   <dfn>effective timeline phase</dfn>
 ::  The associated <a>animation</a>'s <a>hold phase</a> if resolved, otherwise
     the <a>animation</a> timeline's [=timeline phase=].
 
+:   <dfn>at timeline boundary</dfn>
+::  Determined using the following procedure:
+
+    1.  If <em>either</em> of the following conditions are true:
+        *   the associated <a>animation</a>'s [=timeline duration=] is
+            unresolved or zero, or
+        *   the <a>animation</a>'s [=playback rate=] is zero
+
+        return false
+
+    1.  Let <var>effective start time</var> be the <a>animation</a>'s
+        [=start time=] if resolved, or zero otherwise.
+
+    1.  Let <var>effective timeline time</var> be
+        <code><a>animation</a>'s <a>current time</a> / <a>animation</a>'s
+        [=playback rate=] + <var>effective start time</var></code>
+
+    1.  Let <var>effective timeline progress</var> be
+        <code><var>effective timeline time</var> / [=timeline duration=]</code>
+
+    1.  If <var>effective timeline progress</var> is 0 or 1, return true,
+        otherwise false.
+
+Issue: This procedure is not strictly correct for a paused
+animation if the <a>animation</a>'s current time is explicitly set, as this can
+introduce a lead or lag, between the <a>timeline<a>'s current time and
+<a>animation</a>'s current time.
```

Comment:
I'm not sure I understand this issue from the note. It might be helpful to have a specific example, e.g. if a paused animation is set to time 100% the effective timeline time will be Y%.